src/XMakeCommandLine/MSBuild.csproj

```diff
@@ -25,6 +25,7 @@
          two files should be generated from a single source. -->
     <AppConfig Condition="'$(Platform)' == 'x64'">app.amd64.config</AppConfig>
     <ApplicationIcon>MSBuild.ico</ApplicationIcon>
+    <LargeAddressAware>true</LargeAddressAware>
```

Comment:
This was already set in `MSBuildTaskHost`: https://github.com/Microsoft/msbuild/blob/71a52ce09f351fecda8cb598050e491c9d744df7/src/XMakeCommandLine/MSBuildTaskHost/MSBuildTaskHost.csproj#L18-L20